epub33/rs/index.html

```diff
@@ -172,6 +172,13 @@ <h2>EPUB Publications</h2>
 									href="https://www.w3.org/TR/epub-33/#sec-resource-locations">Resource Locations</a>
 								[[!EPUB-33]].</p>
 						</li>
+						<li>
+							<p id="confreq-rs-data-urls">It MUST prevent attemps at top-level navigation to data URLs
```

Comment:
Is this overly restrictive? If a book has an `img` element with a data url, and a reading system has a "open in new window" option for images, would that be disallowed for the data url case?

Reply:
I saw something similar to this in one of the bug trackers cited in the issue. They were going to exempt it from the open option in context menus, so no harm in doing the same here.